pkg/kubelet/kubelet.go

```diff
@@ -1987,31 +1987,32 @@ func (kl *Kubelet) cleanupOrphanedVolumes(pods []*api.Pod, runningPods []*kubeco
 		if _, ok := desiredVolumes[name]; !ok {
 			parts := strings.Split(name, "/")
 			if runningSet.Has(parts[0]) {
-				glog.Infof("volume %q, still has a container running %q, skipping teardown", name, parts[0])
+				glog.Infof("volume %q, still has a container running (%q), skipping teardown", name, parts[0])
 				continue
 			}
 			//TODO (jonesdl) We should somehow differentiate between volumes that are supposed
 			//to be deleted and volumes that are leftover after a crash.
-			glog.Warningf("Orphaned volume %q found, tearing down volume", name)
+			glog.V(3).Infof("Orphaned volume %q found, tearing down volume", name)
 			// TODO(yifan): Refactor this hacky string manipulation.
 			kl.volumeManager.DeleteVolumes(types.UID(parts[0]))
 			// Get path reference count
-			refs, err := mount.GetMountRefs(kl.mounter, cleanerTuple.Unmounter.GetPath())
+			volumePath := cleanerTuple.Unmounter.GetPath()
+			refs, err := mount.GetMountRefs(kl.mounter, volumePath)
 			if err != nil {
-				return fmt.Errorf("Could not get mount path references %v", err)
+				glog.Errorf("Could not get mount path references for %q: %v", volumePath, err)
```

Comment:
This sounds reasonable. I was just asking because the rest of PR involves only logging changes :)
We should make all other cleanup functions to have the same behavior.